### ISSUE TITLE
abnormal retinal cell density terms

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3303]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

1.  abnormal horizontal cell density
2.  increased horizontal cell density
3.  decreased horizontal cell density
4.  abnormal retinal rod bipolar cell density
5.  increased retinal rod bipolar cell density
6.  decreased retinal rod bipolar cell density

>2. Synonyms (e.g., absent spleen)

1.  abnormal retinal horizontal cell density
2.  increased retinal horizontal cell density
3.  decreased retinal horizontal cell density

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

1./2./3.  any change/increase/decrease in the density of the laterally interconnecting neurons in the outer plexiform layer of the retina that connect rods of one part of the retina with cones in another part of the retina
4./5./6  any change/increase/decrease in the density of the bipolar cells that function in low light and transmit signals only through amacrine cells

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

PMID:20127257

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

1.  abnormal horizontal cell morphology (MP:0006068)
4.  abnormal retinal rod bipolar cell morphology (MP:0006074)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0003-3645-6803

**Comment by anna-anagnostop** [comment]
Changed primary term label for all horizontal cell related terms to include "retinal" as part of the label.

Added new terms (with no EQs as we still need a PATO term for cell density):

abnormal retinal horizontal cell density MP:0031186
decreased retinal horizontal cell density MP:0031187
increased retinal horizontal cell density MP:0031188

abnormal retinal rod bipolar cell density MP:0031189
decreased retinal rod bipolar cell density MP:0031190
increased retinal rod bipolar cell density MP:0031191

Used "any anomaly of the density of" and "decrease/increase of the density of" in the textual definitions to be consistent with other existing MP cell density terms.
